cmd/access/node_builder/access_node_builder.go

```diff
@@ -1405,6 +1407,12 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 			"check-payer-balance",
 			defaultConfig.checkPayerBalance,
 			"checks that a transaction payer has sufficient balance to pay fees before submitting it to collection nodes")
+
+		// Register DB Pruning
+		flags.Uint64Var(&builder.registerDBPruneThreshold,
+			"registerdb-pruning-enabled",
```

Comment:
I think `0` should mean unlimited
```suggestion
			"registerdb-pruning-threshold",
```